libcxx/test/std/input.output/file.streams/fstreams/fstream.cons/path.pass.cpp

```diff
@@ -21,11 +21,18 @@
 #include <fstream>
 #include <filesystem>
 #include <cassert>
+#include <type_traits>
+
 #include "test_macros.h"
 #include "platform_support.h"
 
 namespace fs = std::filesystem;
 
+struct fake_path {};
+
+static_assert(std::is_constructible_v<std::fstream, fs::path>);
+static_assert(!std::is_constructible_v<std::fstream, fake_path>);
```

Comment:
It seems that these code compiles even without this pull request because `fake_path` can't be converted to `filesystem::path`: https://godbolt.org/z/1dcsjdeEb

Reply:
Good catch! And thanks for your review! I've refined the test cases, also the title need update.
```
Note: the issue description is now a misnomer, as adding a string_view constructor is no longer being considered at this time.
```